core/server/data/importer/importers/data/base.js

```diff
@@ -19,6 +19,19 @@ class Base {
             returnDuplicates: true
         };
 
+        this.legacyKeys = {};
+        this.legacyMapper = function legacyMapper(item) {
+            item = _.mapKeys(item, function matchLegacyKey(value, key) {
+                return self.legacyKeys[key] || key;
+            });
+
+            item = _.mapValues(item, function alterDefaultLanguage(value, key) {
+                return (key === 'locale' && value === 'en_US') ? 'en' : value;
```

Comment:
This is a bit hard to read, because `locale` is not legacy anymore and we define the keys (language, locale) somewhere else. But i can see why you are doing it.

If you would switch both functions (mapKeys and mapValues), you could do:

```
            item = _.mapValues(item, function alterDefaultLanguage(value, key) {
                if (self.legacyKeys[key]) {
                    if (value === 'en_US') {
                        return null;
                    }
                }

                return value;
            });
```

*note*: default `locale` is null for posts & users.

We could even register the legacy value in the legacy mappings e.g.

```
this.legacy = {
   language: {
      key: 'locale',
      legacyValue: 'en_US'
   }
}
```